lib/utility/bulk_queue.rb

```diff
@@ -41,7 +41,7 @@ def pop_all
     end
 
     def add(operation, payload = nil)
-      raise QueueOverflowError unless will_fit?(operation, payload)
+      raise Errors::BulkQueueOverflowError unless will_fit?(operation, payload)
```

Comment:
maybe some logs here if we are raising? Something like: max bulk size and current payload size and operation?